src/components/DateTimePicker/DateTimePicker.story.jsx

```diff
@@ -8,39 +8,31 @@ import { CARD_SIZES } from '../../constants/LayoutConstants';
 import { getCardMinSize } from '../../utils/componentUtilityFunctions';
 
 // eslint-disable-next-line no-unused-vars
-import DateTimePicker, { INTERVAL_VALUES, RELATIVE_VALUES } from './DateTimePicker';
+import DateTimePicker, {
+  INTERVAL_VALUES,
+  RELATIVE_VALUES,
+  PICKER_KINDS,
+  PRESET_VALUES,
+} from './DateTimePicker';
 
-const customPresets = [
-  {
-    label: 'Last 30 minutes',
-    offset: 30,
+export const defaultRelativeValue = {
+  timeRangeKind: PICKER_KINDS.RELATIVE,
+  timeRangeValue: {
+    lastNumber: 20,
+    lastInterval: INTERVAL_VALUES.MINUTES,
+    relativeToWhen: RELATIVE_VALUES.TODAY,
+    relativeToTime: '13:30',
   },
-  {
-    label: 'Last 1 hour',
-    offset: 60,
-  },
-  {
-    label: 'Last 6 hours',
-    offset: 360,
-  },
-  {
-    label: 'Last 12 hours',
-    offset: 720,
-  },
-];
-
-const defaultRelativeValue = {
-  lastNumber: 20,
-  lastInterval: INTERVAL_VALUES.MINUTES,
-  relativeToWhen: RELATIVE_VALUES.TODAY,
-  relativeToTime: '13:30',
 };
 
-const defaultAbsoluteValue = {
-  startDate: '04/01/2020',
-  startTime: '12:34',
-  endDate: '04/06/2020',
-  endTime: '10:49',
+export const defaultAbsoluteValue = {
+  timeRangeKind: PICKER_KINDS.ABSOLUTE,
+  timeRangeValue: {
+    startDate: '04/01/2020',
```

Comment:
This is out of scope for this PR, but we need to change this so we're using [`ISO 8601`](https://en.wikipedia.org/wiki/ISO_8601) string formats so it will have better interop with moment and other date/time libs.

[Moment docs](https://momentjs.com/docs/#/parsing/string/)

I've opened https://github.com/IBM/carbon-addons-iot-react/issues/1243